test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -193,6 +193,10 @@ export default class BaseFormObject {
     return within(this.el).getByText(text, options);
   }
 
+  getByTextOnScreen(text, options = undefined) {
+    return screen.getByText(text, options);
+  }
+
```

Comment:
Is this just to access `screen` var? Seems redundant...
And `options = undefined` looks unnecessary

Reply:
yeah I made this since the error box is outside the form but may not need it anymore